README.md

```diff
@@ -85,8 +83,6 @@ Install this project with:
 
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
-
 👤 **Joseph David**
```

Comment:
- [ ] Kindly also add your collaborator @juaryimami name and profile links to the Authors section on the Readme file.

Reply:
Thanks, for the review I have implemented the required changes kindly review it again @Omar-Muhamad